packages/server/graphql/types/NotificationSubscriptionPayload.ts

```diff
@@ -34,9 +35,10 @@ const types = [
   CreateTaskPayload,
   DeleteTaskPayload,
   DisconnectSocketPayload,
-  EndCheckInSuccess,
   EndNewMeetingPayload,
+  EndCheckInSuccess,
   EndRetrospectiveSuccess,
+  EndTeamPromptSuccess,
```

Comment:
-1 You're not publishing anything on the `SubscriptionChannel.NOTIFICATION` so there is no need to add the type here.

Reply:
this was kind of future-proofing but you're right, for now there's no need to do it